### ISSUE TITLE
Version/Scan Issue Cleanup for 6 special cases requiring manual editing

### ISSUE THREAD

**Issue by terrywbrady** [CDLUC3/mrt-doc#910]
- ark:/13030/m5f24sdh (**FIXED** - https://merritt.cdlib.org/m/ark%253A%252F13030%252Fm5f24sdh/4)
     **However, two version 3 exist in inv_versions table**
  - ark:/13030/m5f24sdh|3|producer/cubanc_e003598_01.tif
- ark:/13030/m5tb71jh (**FIXED** - https://merritt.cdlib.org/m/ark%253A%252F13030%252Fm5tb71jh/21)
  - ark:/13030/m5tb71jh|1|producer/cubanc_e002828_15.tif
  - ark:/13030/m5tb71jh|5|producer/cubanc_e002828_06.tif   
- ark:/13030/m53j96pp (**FIXED** - https://merritt.cdlib.org/m/ark%253A%252F13030%252Fm53j96pp/6)
  - ark:/13030/m53j96pp|4|producer/cubanc_e004060_05.tif
- ark:/13030/m5dr8mjb (**FIXED** - https://merritt.cdlib.org/m/ark%253A%252F13030%252Fm5dr8mjb/3)
  - ark:/13030/m5dr8mjb|1|producer/ms0414_rec_0605_2.wav    
  - ark:/13030/m5dr8mjb|1|producer/ms0414_rec_0605_ORIG.wav 
- ark:/13030/m5r269bc (**FIXED** - https://merritt.cdlib.org/m/ark%253A%252F13030%252Fm5xh6p0g)
  - ark:/13030/m5r269bc|1|producer/67350632D-B3632752.pdf.
  - ark:/13030/m5r269bc|1|system/mrt-erc.txt   
    - sanitize the metadata to prevent the unicode issue
  - ark:/13030/m5r269bc|1|system/mrt-membership.txt   
  - ark:/13030/m5r269bc|1|system/mrt-owner.txt             
  - remove item from ezid
- ark:/13030/m5v46nq2 (**FIXED** - https://merritt.cdlib.org/m/ark%253A%252F13030%252Fm5v46nq2/31)
  - ark:/13030/m5v46nq2|5|producer/cubanc_e003741_07.tif 

On AWS S3 node 5001
-  ark:/20775/bb21509964 (re-add to object, EL to update afterwards for eScholarship)
   - ark:/20775/bb21509964|3|producer/anvlspec.pdf (**Added** - https://merritt.cdlib.org/m/ark%253A%252F20775%252Fbb21509964/2
  then deleted per subject matter mismatch, see v3)
   - ark:/20775/bb21509964|5|producer/mets.xml (**Not Adding** Metadata is not valid)
- ark:/b6078/d10p4p (investigate which file was targeted for deletion)
   - ark:/b6078/d10p4p|2|system/mrt-delete.txt (**UNKNOWN** - Delete file contains all producer data (see comment below).  Does user want this object deleted?)

On Dryad node 3041
- ark:/13030/m5bc90d2 (was it the intention to delete a file in version 3?)
  - ark:/13030/m5bc90d2|3|mrt-delete.txt

**Comment by mreyescdl** [comment]
@dloy Please download the data in the list to the following host and directory.  It is writable as the "dpr2" role account

Host:          uc3-ingest01x2-stg.cdlib.org
Directory:  /apps/dpr2/tools/node_cleanup_resubmission

**Comment by mreyescdl** [comment]
Object ark:/13030/m5r269bc has a Local ID defined.
We'll need to remove it before we can submit a new object.

```select * from inv_localids where local_id like "%b138009235%";
+---------+---------------------+---------------------+------------+---------------------+
| id      | inv_object_ark      | inv_owner_ark       | local_id   | created             |
+---------+---------------------+---------------------+------------+---------------------+
| 1793778 | ark:/13030/m5r269bc | ark:/13030/j2x634h1 | b138009235 | 2020-07-31 15:09:41 |```
+---------+---------------------+---------------------+------------+---------------------+

**Comment by mreyescdl** [comment]
Removed local ID noted above

```delete from inv_localids where local_id like "%b138009235%";```

**Comment by mreyescdl** [comment]
New object created https://merritt.cdlib.org/m/ark%253A%252F13030%252Fm5xh6p0g

**Comment by mreyescdl** [comment]
Contents of mr-delete.txt file for object "ark:/b6078/d10p4p" for collection UCB Dash

```
AncillaryModelParameters-PSURF.nc
AncillaryModelParameters-BXHEIGHT.nc
AncillaryModelParameters-TP.nc
AncillaryModelParameters-NDENS.nc
BaseCase-DC3_1600-2000.nc
BaseCaseNOProduction.nc
GEOS-Chem-Areas.nc
HNO3Case-DC3_1600-2000.nc
AncillaryModelParameters-TEMP.nc
MPNCase-DC3_1600-2000.nc
HNO3Case-OMI_NO2_1200-1400.nc
N2O5Case-DC3_1600-2000.nc
BaseCase-OMI_NO2_1200-1400.nc
PNACase-DC3_1600-2000.nc
MPNCase-OMI_NO2_1200-1400.nc
README.txt
UpdatedCase-DC3_1600-2000.nc
N2O5Case-OMI_NO2_1200-1400.nc
PNACase-OMI_NO2_1200-1400.nc
UpdatedPlus33Case-DC3_1600-2000.nc
UpdatedCase-OMI_NO2_1200-1400.nc
UpdatedCaseNOProduction.nc
UpdatedPlus33CaseNOProduction.nc
UpdatedPlus33Case-OMI_NO2_1200-1400.nc
```

**Comment by elopatin-uc3** [comment]
@dloy will look into the orphan-copy files that are on node 4001.

**Comment by terrywbrady** [comment]
Rebuilt inv recs for ark:/13030/m5f24sdh and ark:/13030/m5tb71jh.  This resolved the outstanding issues in the con_versions report.

**Comment by elopatin-uc3** [comment]
Moving to Done. Created new ticket for remaining Dryad objects and files under review: #946
